src/Components/Map.jsx

```diff
@@ -110,9 +127,7 @@ class Map extends Component {
 
     return (
 
-      <div id='map'
-        onClick={this.props.sidebarVisible ? () => { this.combineFunctions() } : () => { this.axiosGetPublishedPosts() }}
-      >
+      <div id='map'>
```

Comment:
so we removed the GET requests from map clicks and transferred it to sidebar and legend clicks?